nginx/config.go

```diff
@@ -65,8 +70,16 @@ http {
 		listen 80{{ if $routerConfig.UseProxyProtocol }} proxy_protocol{{ end }};
 		server_name {{$domain}};
 		{{ if $appConfig.Available }}location / {
+			proxy_buffering off;
+			proxy_set_header Host $host;
+			proxy_set_header X-Forwarded-For {{ if $routerConfig.UseProxyProtocol }}$proxy_protocol_addr{{ else }}$proxy_add_x_forwarded_for{{ end }};
+			proxy_redirect off;
+			proxy_connect_timeout 30s;
```

Comment:
Just noticed it isn't for the k8s side but is for any other situation - I'm kinda confused why that is. Maybe someone else knows the legacy reason for that